mysql-py/mysql-conn.py

```diff
@@ -40,8 +34,7 @@ def data_dist ( conn4 ) :
    for row in cur :
        print (row)
 
-print "======Using pymysql========"
-print "     "
+
 import pymysql as my
```

Comment:
`import` should always be on the top of the file. Even though the script has a main function, imports should always go to the top.